tests/descriptor_tests/test_git.py

```diff
@@ -32,8 +34,28 @@ def setUp(self):
 
         # Bare-minimum repo with both annotated and lightweight tags
         self.git_tag_repo_uri = os.path.join(self.fixtures_root, "misc", "tag-test-repo.git")
+
+        # We need to rename the included "git" directory in our tag repo to be
+        # ".git" so that it can be used as a git repo by tests. It'll be named
+        # back to "git" during tearDown.
+        self.git_tag_repo_git = os.path.join(self.git_tag_repo_uri, "git")
```

Comment:
Couldn't you have done like we did on line 31 of the original file with the default config and check in a naked repo?

If you couldn't, I'm not a fan of modifying the repo as unit tests are running. I would either:

Copy the repo to a self.tank_temp sub-directory so that it is cleaned up after the test automatically
OR
If copying the files is too slow then use `self.addCleanup(lambda: shutil.move(...)))` as this will ensure your code is invoked during tear down. This is important because if an exception is thrown by `TankTestBase.tearDown` your code will not be executed and leave the files in the repo in a bad state. And I'm not even sure what will happen if you CTRL-C during the unit tests.